libs/application/templates/financial-aid/src/fields/HomeCircumstancesForm/HomeCircumstancesForm.tsx

```diff
@@ -0,0 +1,128 @@
+import { homeCircumstancesForm } from '../../lib/messages'
+import { useIntl } from 'react-intl'
+
+import { RadioController } from '@island.is/shared/form-fields'
+import { Controller, useFormContext } from 'react-hook-form'
+import { HomeCircumstances } from '@island.is/financial-aid/shared/lib'
+import * as styles from '../Shared.css'
+import cn from 'classnames'
+import { FAFieldBaseProps, InputTypes } from '../../lib/types'
+
+const HomeCircumstancesForm = ({
+  application,
+  field,
+  errors,
+}: FAFieldBaseProps) => {
+  const typeInput = {
+    id: 'homeCircumstances.type',
+    error: errors?.homeCircumstances?.type,
+  } as InputTypes
+
+  const customeInput = {
+    id: 'homeCircumstances.custom',
+    error: errors?.homeCircumstances?.custom,
+  } as InputTypes
+
+  const { formatMessage } = useIntl()
+
+  const { answers } = application
+
+  const [statefulAnswer, setStatefulAnswer] = useState<
+    HomeCircumstances | undefined
+  >(answers?.homeCircumstances?.type)
+
+  const [statefulCustomInput, setstatefulCustomInput] = useState<string>(
+    answers?.homeCircumstances?.custom || '',
+  )
+
+  const { setValue } = useFormContext()
+
+  return (
+    <>
+      <Box marginTop={5}>
+        <RadioController
+          id={`${typeInput.id}`}
+          defaultValue={statefulAnswer}
+          options={[
+            {
+              value: HomeCircumstances.OWNPLACE,
+              label: formatMessage(
+                homeCircumstancesForm.circumstances.OwnPlace,
+              ),
+            },
+            {
+              value: HomeCircumstances.REGISTEREDLEASE,
+              label: formatMessage(
+                homeCircumstancesForm.circumstances.RegisteredLease,
+              ),
+            },
+            {
+              value: HomeCircumstances.UNREGISTEREDLEASE,
+              label: formatMessage(
+                homeCircumstancesForm.circumstances.UnregisteredLease,
+              ),
+            },
+
+            {
+              value: HomeCircumstances.WITHOTHERS,
+              label: formatMessage(
+                homeCircumstancesForm.circumstances.WithOthers,
+              ),
+            },
+            {
+              value: HomeCircumstances.WITHPARENTS,
+              label: formatMessage(
+                homeCircumstancesForm.circumstances.WithParents,
+              ),
+            },
+            {
+              value: HomeCircumstances.OTHER,
+              label: formatMessage(homeCircumstancesForm.circumstances.Other),
+            },
+          ]}
+          onSelect={(newAnswer) =>
+            setStatefulAnswer(newAnswer as HomeCircumstances)
+          }
+          largeButtons
+          backgroundColor="white"
+          error={typeInput.error}
+        />
+      </Box>
+
+      <Box
+        className={cn({
+          [`${styles.inputContainer}`]: true,
+          [`${styles.inputAppear}`]: statefulAnswer === HomeCircumstances.OTHER,
+        })}
+      >
+        <Controller
```

Comment:
yess, we are looking for textarea not Input 😇

Reply:
Ahh so the input component from island.is is multi purpose it is both an input and text area, which breaks single responsibility rule. And the inputcontroller in the application system does not reflect that since it is wrong but ends up in breaking consistency 🤦